### ISSUE TITLE
Contribute anything!

### ISSUE THREAD

**Issue by charmaine** [charmaine/InternetWishlist#3]
Contribute anything you can to this website (a work in progress) & I'll most likely accept your PR as long as it doesn't crash the page! :)

Perfect for beginners (like myself)!

Make sure to read through the README first!

**Comment by maitreyav** [comment]
Amazing! I'm working on it. #2 and this.

**Comment by charmaine** [comment]
@maitreyav I appreciate it. Thanks!

**Comment by hardikyjoshi** [comment]
@charmaine I would like to work on this issue. I have made few updates, but I'm unable to push the changes as I'm not having permissions.

**Comment by charmaine** [comment]
@hardikyjoshi I know you ended up figuring it out since you started the pull request, but just in case anyone encounters the same problem: you have to push to your own forked copy of the repo and start a pull request with your changes, instead of pushing directly to this repo! 👍

**Comment by hardikyjoshi** [comment]
@charmaine  yes, I contributed to one of the first-timers repo and figured the process out. 👍

**Comment by friscomm** [comment]
@charmaine what do you think about storing new colors in local storage? Do you have any other ideas for things I could do to help out or other features you'd like to see on this site? :)

**Comment by maitreyav** [comment]
@friscomm, We are currently thinking to do a transition to PWA. You may also look at #47.  😄
